lib/block-supports/block-style-variations.php

```diff
@@ -299,7 +299,7 @@ function gutenberg_merge_block_style_variations_data( $variations_data, $theme_j
  *
  * @return WP_Theme_JSON_Data_Gutenberg
  */
-function gutenberg_resolve_block_style_variations_from_theme_style_variation( $theme_json ) {
+function gutenberg_resolve_block_style_variations_from_theme_json_user( $theme_json ) {
```

Comment:
Do we need to tweak the docblock to reflect this is more than just a theme style variation as well i.e. it also includes user definitions?